docusaurus/docs/user-docs/releases/introduction.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Introduction to Releases
+description: Introduction to the Releases plugin that enable content managers to group entries to publish/unpublish simultaneously
+---
+
+# Releases <AlphaBadge/> <FutureBadge /> <EnterpriseBadge /> <CloudTeamBadge/>
+
+<SubtleCallout emoji="🧪" title="Experimental feature">
+Releases is currently only available as an experimental, future flag. See <a href="/dev-docs/configurations/features">future flags</a> documentation for instructions on how to enable a future flag.
+</SubtleCallout>
+
+Releases enable content managers to group several entries to publish/unpublish them simultanously. Entries can be grouped by content type or locale, facilitating review and editing before publication.
```

Comment:
> Releases enable content managers to group several entries to publish/unpublish them simultanously.

I think it could be interesting to mention here that these groupings can contain mixed contentTypes, mixed locales, and mixed actions (publish/unpublish)

I would be careful about the word grouping here because eventually grouping will be a way to view the content in a release. 

Maybe we could say something like:

"Releases enables content managers to organize entries into containers that can perform publish and unpublish actions simultaneously. A release can contain entries from different content types and can mix locales."

> Entries can be grouped by content type or locale, facilitating review and editing before publication.

The grouping feature is not implemented for the alpha.